gabimoreno/src/main/java/soy/gabimoreno/data/local/mapper/ToPremiumAudioDbModelMapper.kt

```diff
@@ -15,5 +15,6 @@ fun PremiumAudio.toPremiumAudioDbModel() = PremiumAudioDbModel(
     pubDateMillis,
     audioLengthInSeconds,
     category,
-    excerpt
+    excerpt,
+    hasBeenListened
```

Comment:
Nit:
```suggestion
    hasBeenListened,
```

More info here:
https://gabimoreno.soy/trailing-comma